packages/modules/product/src/services/product-module-service.ts

```diff
@@ -1315,9 +1316,13 @@ export default class ProductModuleService<
       sharedContext
     )) as ProductTypes.CreateProductDTO
 
-    if (!productData.handle && productData.title) {
-      productData.handle = kebabCase(productData.title)
-    }
+    /**
+     * We are already computing handle from title in model. Do we
+     * need here again?
+     */
+    // if (!productData.handle && productData.title) {
+    //   productData.handle = kebabCase(productData.title)
+    // }
```

Comment:
We are already computing handle from title in model. Do we need here again?

Reply:
I would say that we can do it in the service and remove it from the model. My thinking is that in the future the DAL will be replacable, i believe having the feature not tight to the model would be beneficial

Reply:
Sure. I don't have any strong opinions :) So if this the only place that performs CREATE and UPDATE, then we can put the logic in the service directly.